ONSdigital/census-rm-ops-ui
# 1412 r ops UI improve readability of event history

# Motivation and Context

To help view case event history on the r ops ui case details page, the events are now in collapsible boxes which can be expanded to view more details for that specific case event.

# What has changed



- Case events are now in collapsible boxes

# How to test?



- Run the ATs against the branch.
- View the details of some cases and the case events should be in collapsible boxes which can be expanded to view more information.

# Links



[Trello](https://trello.com/c/hvudIE4T/)
